### ISSUE TITLE
More Granular Rotate Events

### ISSUE THREAD

**Issue by marioatmicello** [fnicollet/Leaflet#14]
Currently there is one event which fires inside the setBearing function. In the case of the touchRotate feature, this event fires repeatedly making it hard to react to the event with a single decisive response since it is harder to know when the rotation has completed.

This request for feature is:

Have multiple rotate related events including rotateStart, rotateMove, rotateEnd which fire when the map is being rotated as opposed to just the one event which fires at the end of the setBearing function call.

**Comment by fnicollet** [comment]
That sounds like a nice idea, rotateMove would probably just be "rotate" though. Also, you could use a throttle / debounce function too, depending on what you are trying to achieve

**Comment by marioatmicello** [comment]
True, a throttle / debounce approach would totally work. It would just be really nice to not have to do that and just let the rotation actions dictate the end. Thanks for considering!
